cinema/serializers.py

```diff
@@ -0,0 +1,23 @@
+from rest_framework import serializers
+from cinema.models import Movie
+
+
+class MovieSerializer(serializers.ModelSerializer):
+    id = serializers.IntegerField(read_only=True)
+    title = serializers.CharField(read_only=True)
```

Comment:
The `title` field is set as `read_only=True`, which means it cannot be updated through the serializer. If you intend to allow updates to the `title`, consider removing `read_only=True`.